src/libraries/System.Private.CoreLib/src/System/IO/File.cs

```diff
@@ -731,5 +774,176 @@ private static void Validate(string path, Encoding encoding)
             if (path.Length == 0)
                 throw new ArgumentException(SR.Argument_EmptyPath, nameof(path));
         }
+
+        private static byte[] ReadAllBytesUnknownLength(FileStream fs)
```

Comment:
Did anything change in these methods, or you just moved them between files?

Reply:
I just moved them and I plan to send a separate PR to change them from using `FileStream` to `RandomAccess`

Reply:
Cool